app/views/admin/order_cycles/_exchange_form.html.haml

```diff
@@ -30,9 +30,9 @@
 
 - if type == 'supplier'
   %tr.panel-row{ object: "exchange",
-    panels: "{products: 'exchange_products_supplied'}",
+    panels: "{products: 'exchange_products_supplied', tags: 'exchange_tags'}",
```

Comment:
can you please tell me where the parameter is used?
`tags: 'exchange_tags'`

Reply:
As far as I know, It's used to include the template `app/assets/javascripts/templates/admin/panels/exchange_tags.html.haml` via `app/assets/javascripts/templates/admin/panel.html.haml`